web/src/main/webapp/app/app.js

```diff
@@ -168,6 +175,16 @@
       // Add cache buster interceptor
       $httpProvider.interceptors.push('common.httpRequestInterceptorCacheBuster');
 
+      $httpProvider.interceptors.push(function($window) {
+        return {
+          request: function(config) {
+            if ($window.sessionStorage.getItem('launchId')) {
```

Comment:
See if it is possible to extract the logic to separate method(s) and reuse them in the three places that you add the header.